R/status.R

```diff
@@ -74,18 +127,23 @@ renv_status_impl <- function(project, libpaths, lockpath, sources, cache) {
   if (!ok)
     return(default)
 
-  # get all dependencies, including transitive
-  dependencies <- renv_snapshot_dependencies(project, dev = FALSE)
-  packages <- sort(union(dependencies, "renv"))
-  paths <- renv_package_dependencies(packages, project = project)
-  packages <- as.character(names(paths))
-
   # get lockfile records
   lockfile <- renv_lockfile_records(renv_lockfile_read(lockpath))
 
   # get library records
   library <- renv_snapshot_libpaths(libpaths = libpaths, project = project)
 
+  # get all dependencies, including transitive
+  dependencies <- renv_snapshot_dependencies(project, dev = FALSE)
+  packages <- sort(union(dependencies, "renv"))
+  paths <- renv_package_dependencies(packages, project = project)
+  packages <- as.character(names(paths))
+  # projects will implicitly depend on BiocManager & BiocVersion if any
```

Comment:
This might no longer be necessary; we now implicitly add BiocManager and BiocVersion as dependencies to any Bioconductor packages we find.

Reply:
https://github.com/rstudio/renv/blob/main/R/dependencies.R#L556-L562

Reply:
Yup, confirmed that it's not needed any more (the test we added for this case still passes).